src/Ajax.php

```diff
@@ -361,7 +361,11 @@ public static function createTabs(
          var url_hash = window.location.hash;
          var loadTabContents = function (tablink, force_reload = false, update_session_tab = true) {
             var url = tablink.attr('href');
-            const href_url_params = new URL(url, CFG_GLPI.url_base).searchParams;
+            if (CFG_GLPI.url_base != '') {
+                const href_url_params = new URL(url, CFG_GLPI.url_base).searchParams;
+            } else {
+                const href_url_params = new URLSearchParams($(tablink).prop('href'));
+            }
```

Comment:
Falling back to the current URL origin (i.e. the current scheme + domain) + GLPI base dir would permit to not have two distinct way to retrieve URL parameters.

```suggestion
            var base_url = CFG_GLPI.url_base;
            if (base_url === '') {
                // If base URL is not configured, fallback to current URL domain + GLPI base dir.
                base_url = window.location.origin + '/' + CFG_GLPI.root_doc;
            }
            const href_url_params = new URL(url, CFG_GLPI.url_base).searchParams;
```

Reply:
Seems better, yes.
Maybe default URL should be set to empty string then, no? Rather than current hardcoded value.